app/routes/users/charity_position.js

```diff
@@ -0,0 +1,98 @@
+import AuthorizeRoute from "../authorize";
+import Ember from "ember";
+
+export default AuthorizeRoute.extend({
+  queryParams: {
+    id: {
+      refreshModel: true
+    },
+    organisationId: {
+      refreshModel: true
+    }
+  },
+  organisationsUserService: Ember.inject.service(),
+
+  async model(params) {
+    if (params.id) {
+      return (
+        this.store.peekRecord("organisations_user", params.id) ||
+        (await this.store.findRecord("organisations_user", params.id, {
+          reload: true
+        }))
+      );
+    } else {
+      return this.store.createRecord("organisations_user", {
+        position: "",
+        preferredContactNumber: "",
+        organisationId: "",
+        user_id: "",
```

Comment:
```suggestion
        user_id: this.get('user_id'),
```

just guessing

Reply:
can do that. Its just an initialisation done here

Setting of the user is done in the controller